admin/views/js-templates-primary-term.php

```diff
@@ -14,14 +14,17 @@
 
 <script type="text/html" id="tmpl-primary-term-ui">
 	<?php
+	/* translators: accessibility text. %1$s expands to the term title, %2$s to the taxonomy title. */
+	$yoast_free_js_button_label = __( 'Make %1$s primary %2$s', 'wordpress-seo' );
```

Comment:
```suggestion
	$yoast_seo_js_button_label = __( 'Make %1$s primary %2$s', 'wordpress-seo' );
```
Not sure why the prefix `yoast_free` is introduced here, it's not used anywhere else in the plugin. We use 'yoast_seo' and 'yoast_seo_premium' as far as I know.